src/lustre/lustreTypeChecker.ml

```diff
@@ -1115,7 +1373,10 @@ and check_type_struct_def: tc_context -> LA.eq_lhs -> tc_type -> unit tc_result
                                       ^ " on right hand side of the node equation")
                  else let lhs = List.hd lhss in
                       check_type_struct_item ctx lhs exp_ty)
-    else type_error pos "Cannot reassign value to a constant or enum.")
+    else type_error pos ("Cannot re-assign value to a constant or enum but "
+                         ^ " found reassignmet to identifer(s): "
```

Comment:
Two minor issues:
- Remove one space from the beginning of this string since the previous line ends with a space
- Typo: `reassignmet` -> `reassignment`